<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDlg</class>
 <widget class="QDialog" name="CSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="genSettings">
      <attribute name="title">
       <string>General Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="60,40">
         <item>
          <widget class="QGroupBox" name="groupLanguage">
           <property name="title">
            <string> Language </string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Choose a language: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbxLanguage">
              <item>
               <property name="text">
                <string>de</string>
               </property>
               <property name="icon">
                <iconset resource="vlc-record.qrc">
                 <normaloff>:/flags/de</normaloff>:/flags/de</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ru</string>
               </property>
               <property name="icon">
                <iconset resource="vlc-record.qrc">
                 <normaloff>:/flags/ru</normaloff>:/flags/ru</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>en</string>
               </property>
               <property name="icon">
                <iconset resource="vlc-record.qrc">
                 <normaloff>:/flags/en</normaloff>:/flags/en</iconset>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupLogos">
           <property name="title">
            <string> Logos </string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Reload Logos:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="pushDelLogos">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>36</height>
               </size>
              </property>
              <property name="text">
               <string>&amp;reload Logos</string>
              </property>
              <property name="icon">
               <iconset resource="vlc-record.qrc">
                <normaloff>:/app/reload</normaloff>:/app/reload</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupAccount">
         <property name="title">
          <string> Kartina.tv Account Settings </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="hLayoutUsrPw">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>76</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>76</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>User: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineUsr"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Password: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="linePass">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkAdult">
            <property name="text">
             <string>Allow Adult Channels</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupPath">
         <property name="title">
          <string> Path Settings </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="hLayoutTarget">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>76</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Record Dir:</string>
              </property>
              <property name="margin">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineDir"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushDir">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hLayoutVLCPath">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>76</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>VLC Path:</string>
              </property>
              <property name="margin">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineVLC"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushVLC">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advSettings">
      <attribute name="title">
       <string>Advanced Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupProxy">
         <property name="title">
          <string> Proxy Access </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="useProxy">
            <property name="text">
             <string>Enable Proxy</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hLayoutProxyAddr">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Address: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineProxyHost">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineProxyPort">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hLayoutProxyAcc">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>User: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineProxyUser">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Password: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineProxyPassword">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupFixTime">
         <property name="title">
          <string> Time Correction </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkFixTime">
            <property name="text">
             <string>Try to fix time offset</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupLogging">
         <property name="title">
          <string> Logging </string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Log Level: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbxLogLevel">
            <item>
             <property name="text">
              <string>Log disabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Log Errors only</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Log Errors and Warnings</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Log everything</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupRefresh">
         <property name="title">
          <string> Refresh </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QCheckBox" name="checkRefresh">
              <property name="text">
               <string>Refresh EPG every ...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineInterval">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Minute(s)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutButtons">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushSave">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="vlc-record.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>useProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineProxyHost</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineProxyPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineProxyUser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineInterval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushSave</sender>
   <signal>clicked()</signal>
   <receiver>CSettingsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushCancel</sender>
   <signal>clicked()</signal>
   <receiver>CSettingsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineProxyPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
